### DISCUSSION TITLE
BLDC motor control using Due

### DISCUSSION THREAD

**Discussion by sivaram3291** [askuric/Arduino-FOC#44] [General]
Hi sir,
This is Sivaram, I am developing a double loop cascaded control for BLDC motor drive system. I am having issues with current signal as the current value is very low in at steady state operation my controllers are not working well. Please share experience in this regard.

Thank you

**Comment by askuric** [comment]
Oh, hi @sivaram3291,
I just saw this discussion, I did not receive any notification that you posted something!
I would advise you to post either to the community.simplefoc.com or to the github issues, that way we will see your messages more promptly.
Honestly I did not even know the github discussions exist :D

Ok I would be interested to know your setup?
What kind of driver are you using, what kind of motor?
Is your shunt resistor/amp pair well chosen for the job?

**Comment by sivaram3291** [comment]
Thanks for the reply,

I am using NEMA34 BLDC motor with a rating of 660 watts. I am trying to
control the BLDC motor using Arduino Due. I developed a basic three phase
inverter to drive the motor. For sensing the current used the hall effect
sensors with an offset voltage of 2.5 V.

Now I want to design a new BLDC motor driver board with Due microcontroller
:-) . We can discuss some technical aspects for designing the driver
circuit and the aspects of control algorithms also if needed.

with regards
Chamarthi Sivarama Raju
